src/frontends/pytorch/src/op/rot90.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/transpose.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_rot90(const NodeContext& context) {
+    num_inputs_check(context, 1, 3);
+    auto input = context.get_input(0);
+    int k = context.input_is_none(1) ? 1 : context.const_input<int64_t>(1);
+    std::vector<int64_t> dims = context.input_is_none(2) ? std::vector<int64_t>{0, 1}
```

Comment:
Instead of `const_input` get `dims` as Node using `get_input` and use as is. To avoid requirement for it to be const and to avoid creating a `Constant` with dims later.